### DISCUSSION TITLE
Dịch intractable

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#67] [Terminology] [answered]
Không biết từ này có hay được dịch trong giáo trình không mọi người nhỉ, mình chưa tìm được refs.

Ngữ cảnh:
> the denominator _p_(*x*) is **intractable**, as it requires integrating over all the possible values of *z*, which would require considering all possible combinations of cluster parameters and cluster assignments.

cc @mlbvn/handson-ml2

**Comment by cuongvng** [comment]
context gì anh nhỉ?

**Reply by minhduc0711** [reply]
Mình đã update nhé

**Reply by cuongvng** [reply]
em thấy cũng không phải từ khoá, chắc cứ dịch bình thường cũng được.

**Reply by minhduc0711** [reply]
cứ dịch là `khó tính toán` hả, còn `intractability` thì sao :v

**Reply by ngcthuong** [reply]
Mình thấy có thể dịch theo các từ như: khả đảo - invertable/invertability --> khả tính. Nhưng từ này nghĩa ngược lại với :tính" nên sẽ là không khả tính?

**Reply by cuongvng** [reply]
nếu dùng Hán Việt thì em nghĩ nên để là `bất khả tính`

**Reply by ngcthuong** [reply]
Dùng thuần teen là "ứ khả tính" nha 😁

**Comment by cuongvng** [comment, marked as answer]
chốt `bất khả tính` nhé mn.
